.github/workflows/v2.yml

```diff
@@ -96,26 +95,75 @@ jobs:
       - name: Check that fakes are updated
         run: git diff --no-ext-diff --exit-code
 
-  full-ci:
-    name: V2 CI Check (16.13.2, 1.17.5)
```

Comment:
This name is actually magic - it needs to match https://github.com/weaveworks/corp/blob/master/github-repo-weave-gitops.tf#L50 or you can't merge.

Reply:
good catch